cmd/templates.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+func getUsageTemplate() string {
+	return `Usage:
+  {{.CommandPath}} [command]
+
+Commands:
+  {{rpad "setup" 29}} Setup guide to create your first feature flag
```

Comment:
```suggestion
  {{rpad "setup" 29}} Create your first feature flag using a step-by-step guide
```
just a suggestion - wanted to start this with a verb to match all the other commands

Reply:
👍 makes sense to me!